src/Shared/Resources/xlf/Strings.shared.cs.xlf

```diff
@@ -22,6 +22,11 @@
         <target state="translated">MSB4008: Bylo nalezeno konfliktní sestavení pro sestavení úlohy {0} v umístění {1}.</target>
         <note>{StrBegin="MSB4008: "}UE: This message is shown when the type/class of a task cannot be resolved uniquely from a single assembly.</note>
       </trans-unit>
+      <trans-unit id="DeprecatedEventSerialization">
+        <source>Usage of unsecure BinaryFormatter during serialization of custom event type '{0}'. This will be deprecated soon. Please use Extended*EventArgs instead. More info: https://aka.ms/msbuild/eventargs</source>
```

Comment:
Can we give this a MSBxxxx code so that it can be supressed with `MSBuildWarningsAsMessages`?

Reply:
We already have opt-out mechanism. Since main reason of all those changes is to get rid of BinFmt wherever possible, I feel like there is no need to for easy suppression. In other words, we would rather have them fix it.